dronegen/common.go

```diff
@@ -27,11 +27,6 @@ var (
 		Ref:   triggerRef{Include: []string{"refs/tags/v*"}},
 		Repo:  triggerRef{Include: []string{"gravitational/*"}},
 	}
-	triggerPushMasterOnly = trigger{
-		Event:  triggerRef{Include: []string{"push"}},
-		Branch: triggerRef{Include: []string{"master"}},
-		Repo:   triggerRef{Include: []string{"gravitational/teleport"}},
-	}
```

Comment:
Any particular reason this got moved from a variable to inline?

Reply:
I changed the trigger spec at the one place I was interested in (the push trigger generator). Turns out that this was the only place that used `triggerPushMasterOnly`, so I deleted it.